Python/Product/PythonTools/PythonTools/Intellisense/ProjectAnalyzer.cs

```diff
@@ -381,6 +381,12 @@ out _analysisProcess
                     }
                     entry.SearchPathEntry = childFile.parent;
                     break;
+                case AP.UnhandledExceptionEvent.Name:
+                    Debug.Fail("Unhandled exception from analyzer");
+                    var exception = (AP.UnhandledExceptionEvent)e.Event;
+
+                    VisualStudio.Shell.ActivityLog.LogError(Strings.ProductTitle, exception.message);
```

Comment:
You can use `VsTaskExtensions.ReportUnhandledException` for this (there's a public static non-extension version). This will also Debug.Fail, pop up a dialog, suppress multiple modal reports of the same error, write to the event log, and avoid crashing when the activity log is unavailable.